metric_tank/query_engine.go

```diff
@@ -57,16 +57,17 @@ func alignRequests(reqs []Req, aggSettings []aggSetting) ([]Req, error) {
 	}
 	tsRange := (reqs[0].to - reqs[0].from)
 
+	// note: not all series necessarily have the same raw settings, will be fixed later
```

Comment:
not all series having the same raw settings is already taken into consideration. That is why we use `minInterval`